tests/fixtures/ember-container-query-glint/input/src/components/products/product/card.hbs

```diff
@@ -3,6 +3,7 @@
   @tagName="article"
   data-test-product-card
   local-class="container"
+  ...attributes
```

Comment:
This component covers an edge case of `Element`. The splattributes are passed to a component, which happens to apply them on an `<article>` element. It's hard to guarantee correctness.